src/es/customer-loyality/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="de">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="../../../src/css/variablesCustom.css" rel="stylesheet" type="text/css">
+        <link href="../../../src/css/variablesCustomKlubschule.css" rel="stylesheet" type="text/css">
+        <link href="../../../src/es/components/web-components-toolbox/src/css/colors.css" rel="stylesheet" type="text/css">
+        <script async rel=preload as="script" type="text/javascript"
+            src="../../../src/es/components/web-components-toolbox/src/es/helpers/Environment.js"></script>
+        <title>Customer Portal</title>
+    </head>
+
+    <body>
+        <c-fetch-html>
+            <c-fetch-css>
+                <c-fetch-modules>
+                    <c-translator key="CustomerLoyality">
+                        <c-voting>
+                            <p-general>
+                                <o-body>
+                                    <div>
+                                        <a href="?page=/" route target="_self">
+                                            Voting
+                                        </a>
+                                    </div>
+                                    <main>
+                                        <c-router routes="[
+                                            {
+                                                'name': 'p-voting',
+                                                'path': '../../../customer-loyality/pages/Voting.js',
+                                                'regExp': ['/[#=]{1}\/$/', '/^\/$/']
+                                            }
+                                        ]">
+                                        </c-router>
+                                    </main>
+                                </o-body>
+                            </p-general>
+                        </c-voting>
+                    </c-translator>
+                </c-fetch-modules>
+            </c-fetch-css>
+        </c-fetch-html>
+
+        <script rel="preload" type="module">
```

Comment:
great, now you load all the modules manually. Try to get it running at a second index.html with the wc-config as loader. you may need to adjust the wc-config dictionary (file) with your custom locations.

Reply:
@Weedshaker i'm also loading components inside the webcomponent itself, should i remove this?
![image](https://github.com/mits-gossau/web-components-toolbox-klubschule/assets/2211563/ea2f366c-8915-4abc-8ec1-000fe1efd1e3)

Reply:
nope, those are important to load wc dependencies which a component has within itself. well done. but it is the initial wc imports within index.html, which does wc-config.js for all undefined tags in the light dom when within umbraco.